tests/ast/test_ast_driver.py

```diff
@@ -254,6 +254,8 @@ def run_test(session):
             r"SNOWPARK_TEMP_TABLE_(\w+)", "SNOWPARK_TEMP_TABLE_xxx", raw_unparser_output
         )
         return unparser_output, "\n".join(base64_batches)
+    except Exception as e:
```

Comment:
Explicitly check for exceptions in the `try-finally` block since if an error is raised, it is completely ignored by the code.

Reply:
mabe let's add a comment, or reraise with some meaningful exception?

E.g.,
```
raise Exception('Generated AST test failed') from e
```
or so